src/main/groovy/life/qbic/datamodel/people/Address.groovy

```diff
@@ -86,33 +68,31 @@ class Address  {
    * @return country
    **/
   @JsonProperty("country")
-  public String getCountry() {
+  String getCountry() {
     return country;
   }
 
-  public void setCountry(String country) {
+  void setCountry(String country) {
     this.country = country;
   }
 
-  public Address country(String country) {
+  Address country(String country) {
     this.country = country;
     return this;
   }
 
   @Override
-  public boolean equals(Object a) {
+  boolean equals(Object a) {
     Address ad = (Address) a
     return ad.affiliation.equals(this.affiliation) && ad.country.equals(this.country) && ad.street.equals(this.street) && ad.zipCode == this.zipCode
   }
 
   @Override
-  public String toString() {
+  String toString() {
     StringBuilder sb = new StringBuilder();
     sb.append("class Address {\n");
-
     sb.append("    affiliation: ").append(toIndentedString(affiliation)).append("\n");
     sb.append("    street: ").append(toIndentedString(street)).append("\n");
-//    sb.append("    number: ").append(toIndentedString(number)).append("\n");
     sb.append("    zipCode: ").append(toIndentedString(zipCode)).append("\n");
     sb.append("    country: ").append(toIndentedString(country)).append("\n");
     sb.append("}");
```

Comment:
I think to keep it consistent we could also move to the Annotation and delete the equals method
https://docs.groovy-lang.org/latest/html/api/groovy/transform/EqualsAndHashCode.html

Reply:
Would also save us from 1 or 2 letter variable names ;)

Reply:
Really Good Feedback! Based on it, I introduced a seperate "Address" DTO with a builder pattern. 
After internal discussions, we agreed that it's alright to set the old implementation to deprecated to avoid breaking dependencies down the line and remove it in a future PR.